guide/src/SUMMARY.md

```diff
@@ -32,6 +32,7 @@
   - [Wasm Audio Worklet](./examples/wasm-audio-worklet.md)
   - [web-sys: A TODO MVC App](./examples/todomvc.md)
 - [Reference](./reference/index.md)
+
```

Comment:
Why add the spaces here?
Seems inconsistent with the rest.

Reply:
idk vscode auto-formatter just doing whatever it wants